### PR TITLE
fix #5379

### LINKED ISSUE #5379: Add item lang
### Describe your suggestion

image
image
image

File names:
 "amadron.js" (kubejs\base\recipetypes\pneumaticcraft) 
"fluid_registry.js" ( kubejs\startup_scripts) 
"item_registry.js" (kubejs\startup_scripts)
"multiblock_gamestages.js "(kubejs\startup_scripts)

### Why would you like this added/changed?

It seems that forgot to add lang. Can they be added to crowdin?

### PR DESCRIPTION
**Pull request by ZZZank**
This PR, combined with my previous PRs(like #5435 ), fixes #5379 , with some additions. 

This PR enables localizations for:
- Special custom items
- Multiblock gamestage error message
- Custom Amadron trades

and
- provides corresponding zh_cn localization for new entries
- use `JSON.stringify` to generate nbt string, for a better format
- fixes some wrong translations I made in #5435

### CODE REVIEW

**Comment by ZZZank** [comment]
I also hava a problem: When will Crowdin project gets update? 
I'm asking this because of those translation errors I made in PR4535. They are not severe errors, just missing "bucket" when localizing fluid bucket names, but they're still errors. Translating on Crowdin can largely avoid and qiuckly fix those errors, because there's more participants.

**Comment by ZZZank** [comment]
I also hava a problem: When will Crowdin project gets update? 
I'm asking this because of those translation errors I made in PR4535. They are not severe errors, just missing "bucket" when localizing fluid bucket names, but they're still errors. Translating on Crowdin can largely avoid and qiuckly fix those errors, because there's more participants.

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/pneumaticcraft/amadron.js [hunk 1]
```diff
@@ -10,7 +10,13 @@ onEvent('recipes', (event) => {
                 type: 'ITEM',
                 id: 'pneumaticcraft:reinforced_chest',
                 amount: 1,
-                nbt: `{display:{Name:'[{"text":"Atum Mineral Survey"}]',Lore:['[{"text":"A collection of minerals from Atum.","color":"gold"}]']},BlockEntityTag:{LootTable:"enigmatica:chests/amadron_mineral_survey_atum_combo"}}`
+                nbt: JSON.stringify({
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/pneumaticcraft/amadron.js:13]: Nice change 👌

**Comment by NielsPilgaard** [comment]
Excellent PR as usual, thank you 😊

**Pull request merged**